react/admin/OrganizationsTable.tsx

```diff
@@ -83,9 +83,7 @@ const OrganizationsTable = () => {
           </PageHeaderBottom>
         </PageHeader>
         <PageContent layout="wide">
-          <div className="mb5">
```

Comment:
This div was always some vertical spacing before the table. I moved it within `CheckCustomerSchema` so that this is only added if the alert inside that component is actually rendered. When it's not rendered we shouldn't have that extra spacing on the screen.